docusaurus/docs/cloud/cli/cloud-cli.md

```diff
@@ -51,6 +51,18 @@ Once you deployed your project, if you visit the Strapi Cloud dashboard, you may
 - Some buttons, such as the **Trigger deploy** button, will be greyed out and unclickable since, unless you have [connected a git repository to your Strapi Cloud project](/cloud/getting-started/deployment-cli#automatically-deploying-subsequent-changes).
 :::
 
+## strapi link
```

Comment:
```suggestion
## strapi link <NewBadge />
```

Reply:
I've just added this docs feature today, @gonbaum.
We can now highlight docs section that have been recently updated or are new (see #2151)